src/main/java/duke/task/Deadlines.java

```diff
@@ -0,0 +1,65 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a deadline created by the user that are/will be stored in the database.
+ * A <code>Deadlines</code> inherits from <code>Tasks</code> and is represented by its
+ * name, completion status, and the deadline of the event e.g.,
+ * <code>"Eat lunch", true, "2012-06-03"</code>
+ */
+public class Deadlines extends Tasks {
```

Comment:
Small thing, but perhaps you could change Deadlines to deadline cause it is referring to a single Task

Reply:
I agree.